modules/button/css/lib/icon-button.scss

```diff
@@ -325,14 +326,11 @@
       }
     }
 
-    &.wdc-btn-medium {
-      width: 40px;
-      height: 40px;
-    }
-
     &.wdc-btn-small {
       width: 32px;
       height: 32px;
+      min-width: 32px;
+      max-width: 32px;
```

Comment:
because we're extending `wdc-btn` it's .wdc-btn-small has a min width and max width that is different for icon buttons, so I added this here to override those styles.